### ISSUE TITLE
v2: column "row_id" does not exist

### ISSUE THREAD

**Issue by tim-field** [calebmer/postgraphql#159]
I think this is a bug, but I might be doing something wrong, 

I've got a server running with`classIds: true`

When I try and query for a record by id ( the postgresql table id  ) 

```
query Q {
  allProjects(condition: {rowId: 135}) {
    nodes {
     name 
    }
  }
}
```

I get 

```
{
  "data": {
    "allProjects": null
  },
  "errors": [
    {
      "message": "column \"row_id\" does not exist",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ]
    }
  ]
}
```

( Using id doesn't work either, which isn't surprising as I can see its not available in the ProjectCondition object )

**Comment by calebmer** [comment]
Thanks for reporting 👍

**Comment by calebmer** [comment]
Fixed in `postgraphql@2.0.0-beta.8`.

This is a consequence of how conditions are currently implemented, to fix it I used a quick hack because I intend to improve conditions soonish. There are tests though so we should never regress back to this faulty functionality :+1:
